pkg/cmd/scan.go

```diff
@@ -160,6 +162,13 @@ func NewScanCmd() *cobra.Command {
 		false,
 		"Includes cloud provider service-linked roles (disabled by default)",
 	)
+	fl.BoolVar(&opts.Deep,
+		"deep",
+		false,
+		fmt.Sprintf("%s Enable deep mode, disabled by default\n", warn("EXPERIMENTAL:"))+
```

Comment:
I find it repetitive to say `disabled by default`. If we can enable it, it implicitly means that it is disabled.

Reply:
yeah also Cobra already adds `(default: false)` to the help message